### PR TITLE
Sensors subsystem detail design

### PR DESCRIPTION
**Pull request by DJ-Moye**

### CODE REVIEW

**Review by micahr234 — COMMENTED** [review]
You need to have a teammate review and approve before I review.

**Review by samhunter351 — APPROVED** [review]
Your detail design looks good, a couple quick notes, on line 106 of the markdown file in the hall effect sensor analysis section I think you left the field off of magnetic field, and in your BOM you have a green LED that I'm not sure where comes into play, but these are very small issues, great job!

File: Documentation/Signoffs/Sensors Subsystem Detail Design.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+The IMU and LiDAR will provide positional feedback to the navigation control.
+The photoresistor for the start LED will turn on the robot so that it can start completing its goals.
+This will require activating the controllers, so that they can start running their scripts.
+Each of the controllers will then properly set up and begin running the other subsystems.
+The photoresistors for counting materials and the Hall Effect sensors will send information to the lower-level controller, which is an Arduino Mega.
+This set of sensors will inform the controller about where the materials should be sent.
+
+## Buildable Schematic
+![Sensors Subsystem Schematic](https://github.com/ACruz-42/F24_Team1_CapstoneDemo/blob/aa634817add249cdd91bff9f9b24f9f2ba25e57d/Documentation/Images/Sensors%20Subsystem/Sensors%20Subsystem%20Schematic.svg)
+
```
Comment (micahr234) [inline Documentation/Signoffs/Sensors Subsystem Detail Design.md:60]: Are you planning to build a PCB to connect all of these sensors? If yes you should add it here.

**Review by micahr234 — APPROVED** [review]

File: Documentation/Signoffs/Sensors Subsystem Detail Design.md (hunk 1 above)
Comment (DJ-Moye) [reply Documentation/Signoffs/Sensors Subsystem Detail Design.md:60]: I do not plan on building a PCB. I will use a solderful (or solderable) breadboard.

**Pull request merged**